pkg/database/migrate.go

```diff
@@ -303,3 +305,30 @@ func CreateDefaultExperiment(db *gorm.DB, defaultArtifactRoot string) error {
 	}
 	return nil
 }
+
+// CreateDefaultMetricContext creates the default metric context if it doesn't exist.
+func CreateDefaultMetricContext(db *gorm.DB) error {
+	if err := db.First(&DefaultContext).Error; err != nil {
+		if errors.Is(err, gorm.ErrRecordNotFound) {
+			log.Info("Creating default context")
+			if err := db.Create(&DefaultContext).Error; err != nil {
+				return fmt.Errorf("error creating default context: %s", err)
+			}
+		} else {
+			return fmt.Errorf("unable to find default context: %s", err)
+		}
+	}
+	for _, model := range []interface{}{
+		&Metric{},
+		&LatestMetric{},
+	} {
+		if err := db.Model(model).
+			Where("context_id IS NULL").
+			Update("context_id", DefaultContext.ID).
+			Error; err != nil {
+			return eris.Wrapf(err, "error updating context_id for %t", model)
+		}
+	}
+	log.Debugf("Default metric context: %v", DefaultContext)
```

Comment:
tiny pick -> we use all `log` messages in `lowercase`, at least we try :). can you please make it lowercase.

Reply:
ok